docs/en/loading/Flink-connector-starrocks.md

```diff
@@ -219,6 +220,20 @@ The Flink connector provides the following metrics to monitor loading.
 | totalFlushFailedTimes    | counter | number of times that the data fails to be flushed.                  |
 | totalFilteredRows        | counter | number of rows filtered, which is also included in totalFlushRows.    |
 
+### Flink CDC Synchronization (schema change supported)
+
+[Flink CDC 3.0](https://github.com/ververica/flink-cdc-connectors/releases) framework can be used
+to easily [build a streaming ELT pipeline from CDC sources](https://ververica.github.io/flink-cdc-connectors/master/content/overview/cdc-pipeline.html) (such as MySQL, Kafka) to StarRocks. The pipeline can synchronize whole
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/9392a559-c24d-44c2-ad6b-d3b6609113d4)
Why does the line break at "whole"?